backend/controllers/UserController.ts

```diff
@@ -14,9 +14,11 @@ const UserController = {
         displayName: true,
         description: true,
         imageUrl: true,
-        locations: true,
         subscribers: true,
-        subscriptions: true
+        subscriptions: true,
+        favourite: true,
+        visited: true,
+        personalLocations: true
       });
```

Comment:
why did you rename field locations into personalLocations?

it's obvious that there are only user's locations 🤔